controller/pkg/aclprovider/iptablesprovider.go

```diff
@@ -342,11 +344,19 @@ func (b *BatchProvider) Commit() error {
 	}
 
 	buf, err := b.createDataBuffer()
+
+	if err != nil {
+		zap.L().Error("Failed to create buffer ", zap.Error(err))
+		return fmt.Errorf("Failed to create buffer %s", err)
+	}
+
+	err = b.commitFunc(buf)
+
```

Comment:
The restore function has the error message printed. This will cause duplicate errors. We can restore this back to the original form.

Reply:
ok 👍

Reply:
ok. I'll do that thanks!